toolkit/models.py

```diff
@@ -25,7 +25,7 @@ class Character(models.Model):
     """
 
     Name = models.CharField(max_length=20)
-    AccountOwner = models.ForeignKey(Account, on_delete=models.CASCADE)
+    AccountOwner = models.ForeignKey(User, on_delete=models.CASCADE)

```

Comment:
I belive this should be a OneToOneFiled
[This article](https://simpleisbetterthancomplex.com/tutorial/2016/07/22/how-to-extend-django-user-model.html) goes over how to do it in the section "Extending User Model Using a One-To-One Link"

Reply:
This isn't a one to one relationship it's a one to many because each user can have many characters

Reply:
yeah, I'm talking about the relationship between this model and the User model not between this model and the character model

Reply:
oop sorry